### DISCUSSION TITLE
Set CSS Variable for custom navbar width

### DISCUSSION THREAD

**Discussion by samuelhurni** [Megabit/Blazorise#3440] [Q&A]
Hello Together

I want to change the with of my sidebar when it is collapsed. I was looking how the code is set this width.

I saw, thath this width is set to 64px if the css var "--b-vertical-bar-small-width" is not set. I do not found a place where this variable has been sset so my question is now how can I set this variable manually or where have I set this variable?

Thank you for your help

Samuel

**Comment by David-Moreira** [comment]
Hello,
Please take a look at our Theming docs. These css variables are defined by the `Theme` and you may override them.
You should be able to set these programatically. I believe you should override these options when configuring your theme
**public ThemeBarOptions BarOptions { get; set; }**

https://blazorise.com/docs/theming
